src/extensions/OrderBook.sol

```diff
@@ -1,34 +1,36 @@
 pragma solidity ^0.4.13;
 
-// NOTE: We're using address[100] to get back orderIDs from the orderFetcher as Solidity currently lacks the ability to receive dynamic arrays from external contract calls. Really we should not be returning arbitrary size arrays from internal contracts anywhere. The list gets shrunk to the desired size before returning
+// NOTE: We're using uint256[100] to get back orderIds from the orderFetcher as Solidity currently lacks the ability to receive dynamic arrays from external contract calls. Really we should not be returning arbitrary size arrays from internal contracts anywhere. The list gets shrunk to the desired size before returning
 
 import 'ROOT/libraries/arrays/Uint256Arrays.sol';
 import 'ROOT/Controller.sol';
+import 'ROOT/reporting/Interfaces.sol';
 
 
 // FIXME: Remove this stub once the contract is converted to Solidity
 contract OrdersFetcher {
-    function getOrderIDs(uint256 _type, address _marketID, uint256 _outcomeID, address _startingOrderID, uint256 _numOrdersToLoad) returns (address[100]);
-    function getOrder(address _orderID, uint256 _type, address _marketID, uint256 _outcomeID) returns (uint256[8]);
+    function getOrderIDs(uint256 _type, IMarket _market, uint256 _outcome, uint256 _startingOrderId, uint256 _numOrdersToLoad) returns (uint256[100]);
+    function getOrder(uint256 _orderId, uint256 _type, IMarket _market, uint256 _outcome) returns (uint256[8]);
```

Comment:
Updated in https://github.com/AugurProject/augur-core/commit/366ad0d5a2eee5f42fb9d92961a403d57bb70d74